dmd-script

```diff
@@ -321,7 +321,7 @@ while ( $arg_i < scalar(@ARGV) ) {
 
     if ($arg eq '-arch' ) {
         push @out, '-arch', $ARGV[$arg_i++];
-    } elsif ( $arg =~ m/^-betterC$/ ) {
+    } elsif ( $arg =~ m/^(-betterC|-vcolumns)$/ ) {
```

Comment:
FYI, `-vcolumns` is equivalent to GCC `-fshow-column`, though it's always on in GCC so that mapping would be a no-op.